dll/win32/ws2_32/src/startup.c

```diff
@@ -237,6 +342,30 @@ WSAStartup(IN WORD wVersionRequested,
     /* Check if all worked */
     if (ErrorCode == ERROR_SUCCESS)
     {
+	/* create events for functio WSAThread */
+        gWSAThreadEvent = CreateEvent( 
+		NULL,               // default security attributes
+		TRUE,               // manual-reset event
+		FALSE,              // initial state is nonsignaled
+		TEXT("WSAThreadEvent")  // object name
```

Comment:
Giving this a name means multiple processes will share the same event. That's probably not what you want.